src/lev-report/ListFilter.js

```diff
@@ -0,0 +1,28 @@
+ import React from 'react';
```

Comment:
This extra whitespace should also be removed.

Reply:
Done